tests/Oryx.Tests.Common/DockerVolume.cs

```diff
@@ -42,30 +41,35 @@ private DockerVolume(string originalHostDir, string mountedHostDir, string conta
         /// </summary>
         public string ContainerDir { get; }
 
-        public static DockerVolume Create([NotNull] string hostPath, [NotNull] string containerPath)
+        /// <summary>
+        /// Allows a container to use host's docker engine.
+        /// </summary>
+        /// <returns></returns>
+        public static DockerVolume CreateDockerSocket()
         {
-            if (string.IsNullOrEmpty(hostPath) && !Directory.Exists(hostPath) && !File.Exists(hostPath))
-            {
-                throw new ArgumentException($"'{nameof(hostPath)}' must point to an existing directory or file.");
-            }
-
-            return new DockerVolume(null, hostPath, containerPath);
+            var dockerSocket = "/var/run/docker.sock";
+            return new DockerVolume(originalHostDir: null, dockerSocket, dockerSocket);
         }
 
         /// <summary>
         /// Creates a copy of a local directory, and returns a DockerVolume instance for mounting that copy in a
         /// container.
         /// </summary>
-        /// <param name="originalDir">local directory to be used in a container</param>
+        /// <param name="hostDir">local directory to be used in a container</param>
         /// <returns>DockerVolume instance that can be used to mount the new copy of `originalDir`.</returns>
-        public static DockerVolume CreateMirror(string originalDir)
```

Comment:
I reverted the earlier changes because I do not understand what a 'mirror' here is.

Reply:
a mirror is a directory that is a copy of another directory - that's what this method does:
1. copies source directory to a temporary location
2. mounts that temporary location to the destination path in the container

Reply:
Sure I get it but it doesn't feel something like a term commonly used in docker lingo, agree?

Reply:
It might not be commonly used, but it's OK to introduce terms relevant only to our realm;
we did it with Platform, with Snippet, with Checker, etc.